### PR TITLE
4/8-4/14 Plan

### PR DESCRIPTION
**Pull request by yingren-wang**
HANG out Gang plan for 4/8-4/14

### CODE REVIEW

**Comment by yingren-wang** [comment]
Some helpful information that you may want to check:
- [Issue: When to gym?](https://github.com/dead-programmer/HANG-out/issues/2)
- [Issue: Botanical Gardens still? #Ticket Price#](https://github.com/dead-programmer/HANG-out/issues/3)

**Review by pranmar123 — DISMISSED** [review]
Instead of Botanical Gardens, we can do game board night with the rest of the OPS.

File: April 8-14 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+April 8 - April 14
+
+Monday: TBA (Gym + Movie Night…?)
```
Comment (pranmar123) [inline April 8-14]: Movie Night: Shazam OR Pet Semetary.

@dead-programmer Add the stats of each of the movies in this comment for other reviewers.

File: April 8-14 [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+Tuesday:  Pizzelle's Confections (No break - 4:30-6:00PM chocolate)
+Wednesday: Botanical Garden
+Thursday: Museum of Art ($5 till 8PM)
+Friday: Pane E Vino live concert
+Saturday: VBC ballet show
```
Comment (pranmar123) [inline April 8-14]: Make sure everyone has requested a ballet ticket.

File: April 8-14 (hunk 1 above)
Comment (yingren-wang) [reply April 8-14]: @pranmar123 Why don't you go ahead and find those info as a person who "contributes to initial code base"?

File: April 8-14 (hunk 2 above)
Comment (yingren-wang) [reply April 8-14]: I got two already (don't even know why I got two but I'm fine).

Comment (pranmar123) [reply April 8-14]: @dead-programmer we know why you got two.

Comment (pranmar123) [reply April 8-14]: -_-

Comment (yingren-wang) [reply April 8-14]: @pranmar123 yeah for another Indian friend I know who's in a ballet team

Comment (pranmar123) [reply April 8-14]: interesting invite krystal to the ballet and get her a ticket

**Review by blakepennington97 — COMMENTED** [review]
I vote for #1 Wednesday

**Pull request merged**